user_modules/shared_testcases/testCasesForRAI.lua

```diff
@@ -0,0 +1,169 @@
+        modelYear = "2013",
+        trim = "SE"
+      }
+    })
+  ExpectRequest("VehicleInfo.GetVehicleData", true, { vin = "52-452-52-752" })
+
+  local function button_capability(name, shortPressAvailable, longPressAvailable, upDownAvailable)
+    return
+    {
+      name = name,
+      shortPressAvailable = shortPressAvailable == nil and true or shortPressAvailable,
+      longPressAvailable = longPressAvailable == nil and true or longPressAvailable,
+      upDownAvailable = upDownAvailable == nil and true or upDownAvailable
+    }
+  end
+
+  local buttons_capabilities =
+  {
+    capabilities =
+    {
+      button_capability("PRESET_0"),
+      button_capability("PRESET_1"),
+      button_capability("PRESET_2"),
+      button_capability("PRESET_3"),
+      button_capability("PRESET_4"),
+      button_capability("PRESET_5"),
+      button_capability("PRESET_6"),
+      button_capability("PRESET_7"),
+      button_capability("PRESET_8"),
+      button_capability("PRESET_9"),
+      button_capability("OK", true, false, true),
+      button_capability("SEEKLEFT"),
+      button_capability("SEEKRIGHT"),
+      button_capability("TUNEUP"),
+      button_capability("TUNEDOWN")
+    },
+    presetBankCapabilities = { onScreenPresetsAvailable = true }
+  }
+  ExpectRequest("Buttons.GetCapabilities", true, buttons_capabilities)
+  ExpectRequest("VR.GetCapabilities", true, { vrCapabilities = { "TEXT" } })
+  ExpectRequest("TTS.GetCapabilities", true, {
+      speechCapabilities = { "TEXT", "PRE_RECORDED" },
+      prerecordedSpeechCapabilities =
+      {
+        "HELP_JINGLE",
+        "INITIAL_JINGLE",
+        "LISTEN_JINGLE",
+        "POSITIVE_JINGLE",
+        "NEGATIVE_JINGLE"
+      }
+    })
+
+  ExpectRequest("VR.IsReady", true, { available = true })
+  ExpectRequest("TTS.IsReady", true, { available = true })
+  ExpectRequest("UI.IsReady", true, { available = true })
+  ExpectRequest("Navigation.IsReady", true, { available = true })
+  ExpectRequest("VehicleInfo.IsReady", true, { available = true })
+
+  self.applications = { }
+  ExpectRequest("BasicCommunication.UpdateAppList", false, { })
+  :Pin()
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", { })
+      self.applications = { }
+      for _, app in pairs(data.params.applications) do
+        self.applications[app.appName] = app.appID
+      end
+    end)
+
+  self.hmiConnection:SendNotification("BasicCommunication.OnReady")
+end
+
+--[[@get_data_steeringWheelLocation: read steeringWheelLocation from hmi_capabilities
+--! @parameters: NO
+--]]
+function testCasesForRAI.get_data_steeringWheelLocation()
+  local path_to_file = config.pathToSDL .. 'hmi_capabilities.json'
+  local file  = io.open(path_to_file, "r")
+  local json_data = file:read("*all")
+  
+  file:close()
+
+  local json = require("modules/json")         
+  local data = json.decode(json_data)
+
+  return data.UI.hmiCapabilities.steeringWheelLocation
```

Comment:
@istoimenova In case steeringWheelLocation will be absent in hmi_capabilites.json the script will not start. 
As idea with continious integration is that we create scripts and add to CI for running (it does not matter functionality ready or not in SDL). And when functionality will be implemented tests should be passed. In current case tests even cannot be run when functionality is not implemented yet.